cmake/windows-setup.cmake

```diff
@@ -49,6 +49,7 @@ if(FLB_WINDOWS_DEFAULTS)
   set(FLB_IN_STATSD             Yes)
   set(FLB_IN_STORAGE_BACKLOG    Yes)
   set(FLB_IN_EMITTER            Yes)
+  set(FLB_IN_ELASTICSEARCH       No)
```

Comment:
make it available by default

Reply:
I tried to enable this plugin on Windows but I wasn't able to be usable.
This is because registering HTTP server based plugin is not runnable and usable even if specifying `On`.
`in_http` and `in_opentelemetry` are also same circumstances. Am I wrong?

Reply:
Actually, I was wrong. :cry:

Fixed in https://github.com/fluent/fluent-bit/pull/6685/commits/a2d42f23f3a328194e72ebebea0d7df10e943ad5 and https://github.com/fluent/fluent-bit/pull/6685/commits/0f97dc1f3d7797c6a9fd5099a9f9cb1b5d76d16f.